lib/compress/zstd_compress_internal.h

```diff
@@ -337,6 +349,8 @@ struct ZSTD_CCtx_s {
 #if ZSTD_TRACE
     ZSTD_TraceCtx traceCtx;
 #endif
+
+    size_t alignmentBytes;     /* Size of extra object reserved to align the workspace's tables to 64-bytes */
```

Comment:
Is this still necessary with the cwksp changes?

Reply:
Nope! Was gonna remove once I can rebase this.